### ISSUE TITLE
Milestone 2e Accessibility & implementation of profile editor

### ISSUE THREAD

**Issue by timea-solid** [solid/solidos#233]
Profile Pane has an [edit Pane](https://github.com/SolidOS/profile-pane/tree/main/src/editProfilePane) which at the moment is created from a Form. We would like to move away from RDF forms for the profile edit which means the profile edit has to be redone.
Part of the reimplementation should be:

- [x] extracting all inline CSS in dedicated CSS files (example in [profile-pane](https://github.com/SolidOS/profile-pane/tree/main/src/styles)). Very important is to create dedicated CSS pane names. Example: if you use the .warning, its CSS will be coming from [mash.css](https://github.com/SolidOS/mashlib/blob/3e859f55ac738c6c4b2a3bc2310d903ae6ab797b/src/styles/mash.css#L77C1-L77C10) so create a dedicated .warning-contact for it if the one in mash.css does not fit.
- [x] making sure the CSS is part of release (example of location mention in [webpack](https://github.com/SolidOS/profile-pane/blob/022cb2458371efdee74d781d7203703d4f524ac1/webpack.config.mjs#L61)) and according entry file (example [index.ts](https://github.com/SolidOS/profile-pane/blob/022cb2458371efdee74d781d7203703d4f524ac1/src/index.ts#L3))
- [x] using already available CSS variables from [mashlib light theme ](https://github.com/SolidOS/mashlib/blob/main/src/styles/themes/light.css) and documenting in the PR what other CSS variables could be moved to the main mashlib theme

- [x] adding HTML5 and ARIA labels
- [x] making sure it also work when resizing screen
- [ ] add accessibility tests based on 'axe-core' (example [here](https://github.com/SolidOS/profile-pane/blob/main/test/cv-card.accessibility.test.ts))
- [x] making sure other accessibility features are inline with the WCAG standard (the tests will show). As example: text sizes,  button sizes, keyboard focus and so on.

**Comment by timea-solid** [comment]
all in Mashlib 2.1.5
